documentation/features/forecasting.rst

```diff
@@ -62,6 +62,88 @@ Note that:
 ``forecast-frequency`` together with ``max-forecast-horizon`` determine how the forecasting cycles advance through time.
 ``start-date`` / ``from-date`` and ``to-date`` allow precise control over the training and prediction windows in each cycle.
 
+Forecasting via the API
+-----------------------
+
+In addition to the CLI command, FlexMeasures now provides API endpoints for
+triggering forecasts and retrieving their results.
+
+These endpoints live under the Sensor API (``/api/v3_0/sensors``).
+
+Triggering a forecast
+^^^^^^^^^^^^^^^^^^^^^
+
+.. http:post:: /api/v3_0/sensors/<sensor>/forecasts/trigger
```

Comment:
`<sensor>` -> `<id>`

We could also try adding a cross-reference here to the endpoint description in the API v3 docs. Maybe some of the contents should move there, actually.